wade-review.md

```diff
@@ -0,0 +1,596 @@
+## Getting started with Kafka Streams: The Processing Process
+
+
+Now, I'll pivot to building a Kafka Streams application that processes the 'github-pull_requests' events, creating an up-to-date ratio of open/closed pull requests.
+
+If you want to follow along with this part, check out the GitHub repo in the links below. I’m going to focus on the domain logic given our timeframe, so if you really want to dive in, I suggest consulting the codebase. 
+
+
+
+```java
+static class MyProcessorSupplier implements ProcessorSupplier<String, GitHubPRInfo, String, GitHubPRStateCounter> {
+}
+```
+
+Now I'll  write the ProcessorSupplier, using the class provided by KafkaStreams. I'll use the record shape supplied by GitHubPRInfo as well as the object provided by GitHubPrStateCounter in our models folder.
+
+---
+
+```java
+static class MyProcessorSupplier implements ProcessorSupplier<String, GitHubPRInfo, String, GitHubPRStateCounter> { 
+	static final String STORE_KEY = "state-counter";
+}
+```
+
+Our store key will be named 'state_counter'. 
```

Comment:
What is a store key?

Reply:
And why are you calling it `state_counter`? What state is being counted?

Reply:
https://github.com/Cerchie/notes-for-gh-video/pull/2/commits/736558519b28246eceeda927a8e5c2d9508470e5